packages/admin/package.json

```diff
@@ -15,53 +15,30 @@
     "postinstall": "node ./postinstall.js"
   },
   "dependencies": {
-    "@auth0/auth0-react": "^1.4.0",
     "@aws-crypto/client-node": "3.2.0",
-    "@contentful/rich-text-from-markdown": "15.16.6",
     "@contentful/rich-text-react-renderer": "^15.12.1",
-    "@elastic/elasticsearch": "7.10",
     "@tinymce/tinymce-react": "4.3.2",
     "axios": "^0.27.2",
-    "body-parser": "^1.19.2",
-    "contentful": "^9.1.5",
     "cookie-parser": "^1.4.6",
     "csurf": "^1.11.0",
-    "cypress-recurse": "^1.18.0",
-    "date-fns": "^2.19.0",
-    "dotenv": "^16.0.0",
     "fs-extra": "11.2.0",
     "gap-web-ui": "*",
     "govuk-frontend": "4.7.0",
-    "gray-matter": "4.0.3",
-    "imap-simple": "^5.1.0",
-    "jsonwebtoken": "^9.0.0",
-    "mailparser": "^3.4.0",
     "moment": "^2.29.4",
-    "moment-timezone": "^0.5.37",
     "next": "12.3.4",
-    "next-logger": "^3.0.1",
```

Comment:
See comment [here](https://github.com/cabinetoffice/gap-find-web/pull/102#discussion_r1440510526) - could you leave this one in?

Reply:
Yup, adding back in all the ones that shouldn't have come out now 🥲